src/components/Live/LiveProvider.js

```diff
@@ -9,6 +9,7 @@ function LiveProvider({
   code,
   language,
   theme,
+  enableTypeScript = true,
```

Comment:
We should remove this, since we have `defaultProps.enableTypeScript = true`  (and none of the other props have defaults)

Reply:
The reason we have this is because TypeScript will think it's a required prop when imported directly as the project is mixed. The next phase of this will be converting the remaining source over to TypeScript and dropping PropTypes.

Reply:
Ah, gotcha!  ✅